user/tests/wiring/no_fixture_long_running/cellular.cpp

```diff
@@ -114,7 +114,55 @@ test(CELLULAR_02_device_will_connect_to_the_cloud_when_all_udp_sockets_consumed)
     // Then the device overcomes this socket obstacle and connects to the Cloud
     assertEqual(Particle.connected(), true);
 }
-
 #endif // !HAL_USE_SOCKET_HAL_POSIX
 
+test(CELLULAR_03_device_will_reconnect_to_the_cloud_when_unexpected_modem_brown_out_occurs_within_2min) {
+#if PLATFORM_ID != PLATFORM_ELECTRON
+    skip();
+    return;
+#endif
+    // Serial.println("the device will reconnect to the cloud when unexpected modem brown out occurs within 2min");
+    // Given the device is currently connected to the Cloud
+    connect_to_cloud(6*60*1000);
+    // When the device modem suffers a simulated brown out (modem reset and powered back on with no initialization)
+    digitalWrite(RESET_UC, 0);
+    delay(10000);
+    digitalWrite(RESET_UC, 1);
+    do {
+        // SARA-U2/LISA-U2 50..80us
+        digitalWrite(PWR_UC, 0); delay(50);
```

Comment:
Nit: single statement on a single line. It's very easy to miss things with this pattern.